### ISSUE TITLE
still active?

### ISSUE THREAD

**Issue by phishy** [filefog/filefog#10]
Are you still working on this? Is seems the last commit was in February. Is there another similar project. I was thinking I might take a stab at it for use in https://github.com/phishy/onedrop.

**Comment by AnalogJ** [comment]
Hey @phishy 
I've used it before off and on for a few different projects, and it should still work. There's not much maintenance required as it uses the third party libraries under the hood to do most of the heavy lifting, and filefog just adds a translation layer on top. 


Recently I've tried looking at using [cloudrail](https://github.com/CloudRail/cloudrail-si-node-sdk) for a new project, but I couldn't get it working. 
It wasn't working in AWS Lambda, but that might not be a problem for you.
